setup1-shared/src/reliability/message.rs

```diff
@@ -0,0 +1,126 @@
+use std::io::{BufReader, Read};
+
+use tokio::io::{AsyncRead, AsyncReadExt, AsyncWrite, AsyncWriteExt};
+
+type Error = Box<dyn std::error::Error + Send + Sync>;
+
+pub const MAXIMUM_MESSAGE_SIZE: usize = 101 * 1024 * 1024; // 101 Mb
```

Comment:
guessing this is in bytes? might be good to mention the unit here

Reply:
yes, bytes